tests/test_integration/test_stencil_object.py

```diff
@@ -64,3 +65,12 @@ def runit(*args, **kwargs) -> float:
     assert len(stencil._domain_origin_cache) == 0
     cleaned_cache_time = runit(in_storage, out_storage, offset=1.0)
     assert cleaned_cache_time > fast_time
+
+
+@pytest.mark.parametrize("backend", ALL_BACKENDS)
+@pytest.mark.parametrize("backend_opts", [{"verbose": True}])
+def test_common_stencil_options(backend, backend_opts):
+    @gtscript.stencil(backend=backend, **backend_opts)
+    def foo(f: Field[float]):
+        with computation(PARALLEL), interval(...):  # type: ignore
+            f = 42.0  # noqa F841
```

Comment:
Is this testing that the `verbose` option is allowed in all backends, even if it is not defined as an option in some? Can you capture the warning and assert that it happens?

Reply:
I changed the test to actually test to use a random option that none of the backends supports and check for the warning. I think it documents better the purpose of this change.